components/core/tools/scripts/lib_install/ubuntu-jammy/install-prebuilt-packages.sh

```diff
@@ -15,9 +15,6 @@ DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
   build-essential \
   git \
   jq \
-  libboost-filesystem-dev \
-  libboost-iostreams-dev \
-  libboost-program-options-dev \
```

Comment:
Chose to switch to the boost install script on jammy because libboost-url-dev is only available from ubuntu noble onwards.

Reply:
How about the MacOS install scripts?

Reply:
We use the brew install formula for boost, which appears to be on 1.86 right now, so it satisfies the requirement.

https://formulae.brew.sh/formula/boost